mantle/platform/api/aws/images.go

```diff
@@ -329,15 +329,22 @@ func (a *API) CreateImportRole(bucket string) error {
 	return nil
 }
 
-func (a *API) CreateHVMImage(snapshotID string, diskSizeGiB uint, name string, description string) (string, error) {
+func (a *API) CreateHVMImage(snapshotID string, diskSizeGiB uint, name string, description string, architecture string) (string, error) {
 	var awsArch string
-	switch runtime.GOARCH {
+	if architecture == "" {
+		architecture = runtime.GOARCH
+	}
+	switch architecture {
 	case "amd64":
+		fallthrough
+	case "x86_64":
```

Comment:
Instead of the fallthrough, you can use:
````
case "amd64", "x86":
   ...
```

But that's a nit.

Reply:
fixed in latest upload